### PR TITLE
Removed some todo-lev comments

### PR DESCRIPTION
**Pull request by samgermain**

### CODE REVIEW

File: freqtrade/exchange/bybit.py [hunk 1]
```diff
@@ -26,7 +26,6 @@ class Bybit(Exchange):
 
     _supported_trading_mode_collateral_pairs: List[Tuple[TradingMode, Collateral]] = [
         # TradingMode.SPOT always supported and not required in this list
-        # TODO-lev: Uncomment once supported
```
Comment (samgermain) [inline freqtrade/exchange/bybit.py:29]: I just left the one in binance.py

File: freqtrade/freqtradebot.py [hunk 2]
```diff
@@ -68,7 +68,6 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         init_db(self.config.get('db_url', None), clean_open_orders=self.config['dry_run'])
 
-        # TODO-lev: Do anything with this?
```
Comment (samgermain) [inline freqtrade/freqtradebot.py:71]: only for cross margin, there's a trello task about it

File: freqtrade/freqtradebot.py [hunk 3]
```diff
@@ -1128,7 +1127,6 @@ def handle_cancel_enter(self, trade: Trade, order: Dict, reason: str) -> bool:
         Buy cancel - cancel order
         :return: True if order was fully cancelled
         """
-        # TODO-lev: Pay back borrowed/interest and transfer back on leveraged trades
```
Comment (samgermain) [inline freqtrade/freqtradebot.py:1136]: Created a trello task with the margin label

File: tests/optimize/test_hyperopt.py [hunk 4]
```diff
@@ -22,9 +22,6 @@
                             patched_configuration_load_config_file)
 
 
-# TODO-lev: This file
```
Comment (samgermain) [inline tests/optimize/test_hyperopt.py:25]: To be done with hyperopt updates

File: tests/test_freqtradebot.py [hunk 5]
```diff
@@ -2951,7 +2951,7 @@ def test_execute_trade_exit_down(default_conf_usdt, ticker_usdt, fee, ticker_usd
 @pytest.mark.parametrize(
     "is_short,amount,open_rate,current_rate,limit,profit_amount,profit_ratio,profit_or_loss", [
         (False, 30, 2.0, 2.3, 2.25, 7.18125, 0.11938903, 'profit'),
-        (True, 29.70297029, 2.02, 2.2, 2.25, -7.14876237, -0.11944465, 'loss'),  # TODO-lev
```
Comment (samgermain) [inline tests/test_freqtradebot.py:2964]: I think this is outdated

**Review by xmatthias — APPROVED** [review]

**Pull request merged**